app_tests/integration_tests/llm/cpu_llm_server_test.py

```diff
@@ -82,9 +82,18 @@ def test_llm_server(llm_server, available_port):
     # Here you would typically make requests to your server
     # and assert on the responses
     assert llm_server.poll() is None
-    output = do_generate("1 2 3 4 5 ", available_port)
-    logger.info(output)
+    PROMPT = "1 2 3 4 5 "
     expected_output_prefix = "6 7 8"
+    logger.info("::group::Sending HTTP Generation Request")
```

Comment:
Was this also supposed to create it's own group?

Looks like it needs an `::endgroup` at the end, with logging the output:

```text
INFO     app_tests.integration_tests.llm.cpu_llm_server_test:cpu_llm_server_test.py:87 ::group::Sending HTTP Generation Request
INFO     app_tests.integration_tests.llm.cpu_llm_server_test:cpu_llm_server_test.py:35 Generating request...
INFO     app_tests.integration_tests.llm.cpu_llm_server_test:cpu_llm_server_test.py:48 Prompt text:
INFO     app_tests.integration_tests.llm.cpu_llm_server_test:cpu_llm_server_test.py:49 1 2 3 4 5 
INFO     app_tests.integration_tests.llm.cpu_llm_server_test:cpu_llm_server_test.py:52 Generate endpoint status code: 200
INFO     app_tests.integration_tests.llm.cpu_llm_server_test:cpu_llm_server_test.py:54 Generated text:
INFO     app_tests.integration_tests.llm.cpu_llm_server_test:cpu_llm_server_test.py:96 6 7 8 9 10 11 12
```